tests/proc_macros.rs

```diff
@@ -0,0 +1,50 @@
+use salsa::Database;
+use std::sync::Arc;
+
+/// My awesome db trait.
```

Comment:
Seems like at this point this doesn't test anything that isn't covered by other tests, right?

Reply:
Good point, I'll remove this.